lms/djangoapps/certificates/management/commands/fix_ungraded_certs.py

```diff
@@ -52,8 +52,8 @@ def handle(self, *args, **options):
 
         for cert in ungraded:
             # grade the student
-            grade = course_grades.summary(cert.user, course)
-            print "grading {0} - {1}".format(cert.user, grade['percent'])
-            cert.grade = grade['percent']
+            grade = CourseGradeFactory().create(cert.user, course)
+            print "grading {0} - {1}".format(cert.user, grade.percent)
```

Comment:
I suppose they were there since this an older management command.  I'll change them to log statements.